components/ntp_sponsored_images/browser/locale_helper.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_NTP_SPONSORED_IMAGES_BROWSER_LOCALE_HELPER_H_
+#define BRAVE_COMPONENTS_NTP_SPONSORED_IMAGES_BROWSER_LOCALE_HELPER_H_
+
+#include <string>
+
+#include "base/macros.h"
+#include "base/memory/singleton.h"
+
+namespace internal {
+
+const char kDefaultLocale[] = "en-US";
+
+class LocaleHelper {
+ public:
+  static LocaleHelper* GetInstance();
+
+  void set_for_testing(
+      LocaleHelper* locale_helper);
+
+  // Should return the language based upon the tagging conventions of RFC 4646
+  virtual const std::string GetLocale() const;
+
+  LocaleHelper(const LocaleHelper&) = delete;
```

Comment:
use DISALLOW_COPY_AND_ASSIGN

Reply:
OK to leave as-is, IMO. See note at https://github.com/brave/brave-core/pull/4075/files/2f7f51f8a0ffee135d823311f7bb464dd74e6acc#r370352211